### ISSUE TITLE
Feature Request: Allow Multiple Sensor Recording

### ISSUE THREAD

**Issue by kevinfodor** [kevinfodor/my-sensors#11]
This is from an app user who had a suggestion:

"Another recommendation, if you are still interested in improving on what you have already, would be to allow selecting multiple sensors for recording and possibly adding the GPS to the list of record-able items.
To do this add a checkbox next to each sensor on the homepage.
The user could go into each sensor and change its settings as one can now and underneath the bottom of the homepage is a  button [Log all checked Sensors] 
While logging you just create an additional outer loop that goes through all sensors and adds the value from the checked ones to the file.
I see that this becomes tricky when some sensors update very fast and others only rarely. 

Then one could strap the phone to something that moves and get two independent acceleration measurements, multiple rotation  and heading readings and if needed the changing illumination.

Think model train or race car movements. Roller-coaster ride and road-trips not only with geo-location but multiple, parallel, time-stamped exact sensor data. one could integrate accelerations to get velocities and integrate those to get relative location. 

There are already multiple GPS track recorders that one could let run in parallel and hopefully it will be possible to align the time stamps, so recording GPS would be for me the lower priority."
